### PR TITLE
fix(bil): requirements update

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Updated IAM + requirements banners.

### CODE REVIEW

**Review by nerda-codes — APPROVED** [review]

File: console/billing/how-to/add-billing-contact.mdx [hunk 1]
```diff
@@ -13,11 +13,11 @@ categories:
...
 
-The **billing contact** is an additional contact linked to your account who will receive your monthly invoices. This can be useful if invoices in your Organization are handled by another department. 
+**Billing contact** is an additional contact linked to your account who will receive your monthly invoices. This can be useful if invoices in your Organization are handled by another department.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```
Comment (RoRoJ) [inline console/billing/how-to/add-billing-contact.mdx:20]: I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?

File: console/billing/how-to/change-billing-information.mdx [hunk 2]
```diff
@@ -13,9 +13,10 @@ categories:
   - billing
 ---
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```
Comment (RoRoJ) [inline console/billing/how-to/change-billing-information.mdx:18]: I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?

File: console/billing/how-to/change-payment-method.mdx [hunk 3]
```diff
@@ -15,22 +15,22 @@ categories:
...
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-  - You have added a valid credit card on your account, even if you have set up SEPA as the default payment method.
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- A valid credit card added to your account, even if you have set up SEPA as the default payment method.
+
```
Comment (RoRoJ) [inline console/billing/how-to/change-payment-method.mdx:22]: I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?

File: console/billing/how-to/download-an-invoice.mdx [hunk 4]
```diff
@@ -13,9 +13,10 @@ categories:
   - billing
 ---
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+
```
Comment (RoRoJ) [inline console/billing/how-to/download-an-invoice.mdx:19]: I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?

File: console/billing/how-to/redeem-voucher-code.mdx [hunk 5]
```diff
@@ -17,10 +17,10 @@ A Scaleway voucher provides you with a discount or credit to use on our services
...
 Voucher codes are unique and valid only for a defined period of time.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-  - You have a valid payment method added to your account.
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- A valid payment method added to your account.
```
Comment (RoRoJ) [inline console/billing/how-to/redeem-voucher-code.mdx:23]: I think you might also need IAM permissions to add a billing contact, so we should add the requirement about being the Owner or having appropriate permissions- @ldecarvalho-doc  can you confirm?

File: console/billing/how-to/add-billing-contact.mdx (hunk 1 above)
Comment (ldecarvalho-doc) [reply console/billing/how-to/add-billing-contact.mdx:20]: yes we have to add the iam req 🙏🏻 
the Iam requirement should be included in all pages except certain IAM pages hehe (and some account pages too)

Comment (jcirinosclwy) [inline console/billing/how-to/add-billing-contact.mdx:20]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: console/billing/how-to/change-billing-information.mdx (hunk 2 above)
Comment (jcirinosclwy) [inline console/billing/how-to/change-billing-information.mdx:18]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: console/billing/how-to/change-payment-method.mdx [hunk 6]
```diff
@@ -15,22 +15,22 @@ categories:
 
 If you have registered both a **credit card** and a **SEPA mandate**, you can choose the default payment method you wish to use for your Scaleway account.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-  - You have added a valid credit card on your account, even if you have set up SEPA as the default payment method.
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline console/billing/how-to/change-payment-method.mdx:20]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

File: console/billing/how-to/download-an-invoice.mdx [hunk 7]
```diff
@@ -13,9 +13,10 @@ categories:
   - billing
 ---
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline console/billing/how-to/download-an-invoice.mdx:18]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**